api/main/__init__.py

```diff
@@ -23,34 +24,37 @@ def get_env_variable(name: str) -> str | None:
         message = f"Expected env variable '{name}' not set."
         raise Exception(message) from exception
 
+
 # https://github.com/miguelgrinberg/Flask-SocketIO-Chat
 app = Flask(__name__)
-print( get_env_variable("SQLALCHEMY_DATABASE_URI") )
+print(get_env_variable("SQLALCHEMY_DATABASE_URI"))
 # loads .env file into the envrionment correctly.
 load_dotenv()
-app.config['SQLALCHEMY_DATABASE_URI'] = os.environ.get( "SQLALCHEMY_DATABASE_URI" )
-db = SQLAlchemy( app )
-from .models import * 
-migrate = Migrate( app, db )
+app.config["SQLALCHEMY_DATABASE_URI"] = os.environ.get("SQLALCHEMY_DATABASE_URI")
+db = SQLAlchemy(app)
+from .models import *
+
+migrate = Migrate(app, db)
 app.config["SECRET_KEY"] = "secret!"
 CORS(app, resources={r"/*": {"origins": "*"}})
 sio = SocketIO(app, cors_allowed_origins="*")
 
 
 
-
 # Set variables from .env to global scope
 POSTGRES_URL = get_env_variable("POSTGRES_URL")
 POSTGRES_USER = get_env_variable("POSTGRES_USER")
 POSTGRES_PW = get_env_variable("POSTGRES_PW")
 POSTGRES_DB = get_env_variable("POSTGRES_DB")
+DATA_FILEPATH = get_env_variable("DATA_FILEPATH")
 
+from .file_writing.views import file_writer
 from .observations.views import observations
-from .logsheet.views import logsheet
```

Comment:
This line is removed but never added back, preventing the back-end server from starting. I'll push a change.